src/index.html

```diff
@@ -11,7 +11,134 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="fieldset">
+        <legend class="leg">Personal information</legend>
+        <label class="form">
+          Surname:
+          <input
+           type="text"
+           name="surname"
+           autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
             type="text"
             name="surname"
             autocomplete="off"
          >
```
fix indentations everywhere